devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -122,10 +122,17 @@ private Map parseCatchAllMessage(String description) {
 				break
```

Comment:
Since command 0x07 isn't ignored anymore you need to make a change to the other case in this switch statement so that it only handles the correct commands. One option is just make sure the command is not 0x07